docs/001_develop/03_client-capabilities/008_interaction/date-picker.mdx

```diff
@@ -0,0 +1,283 @@
+            <td>visible</td>
+            <td><code>boolean</code></td>
+            <td>Displays the calendar by default. Defaults to <code>false</code></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker visible>
+            ```
+
+            </td>
+        </tr>
+        <tr>
+            <td>inline-calendar</td>
+            <td><code>boolean</code></td>
+            <td>Displays the calendar as an inline block element as opposed to being absolutely positioned relative to it's parent. Defaults to <code>false</code></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker inline-calendar>
+            ```
+
+            </td>
+        </tr>
+        <tr>
+          <td>hide-weekends</td>
+          <td><code>boolean</code></td>
+          <td>Hides Saturday and Sundays restricting user to selecting only week days. Defaults to <code>false</code></td>
+          <td>
+            ```typescript
+            <rapid-date-picker hide-weekends>
+            ```
+          </td>
+        </tr>
+        <tr>
+          <td>position</td>
+          <td><code>string</code></td>
+          <td>Determines the position of the calendar. Possible options 'right', 'left', 'center' or 'left'.</td>
+          <td>
+            ```typescript
+              <rapid-date-picker position="right">
+            ```
+          </td>
+        </tr>
+        <tr>
+          <td>label</td>
+          <td><code>string</code></td>
+          <td>Sets the label that will be displayed next to the date input text field</td>
+          <td>
+            ```typescript
+              <rapid-date-picker label="Trade Date">
+            ```
+          </td>
+        </tr>
+        <tr>
+          <td>value</td>
+          <td><code>string</code></td>
+          <td>Sets the date value as string in the configured format on the date picker.</td>
+          <td>
+            ```typescript
+              <rapid-date-picker label="06-20-2025">
+            ```
+          </td>
+        </tr>
+      </tbody>
+  </table>
+
+### Properties
+
+<table>
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Type</th>
+      <th>Use</th>
+      <th>Example</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>disabledDaysOfWeek</td>
+      <td><code>() => [2,3]</code></td>
+      <td>An array of numbers disabling days of the week with 0 being Sunday and 6 being Saturday</td>
+      <td>
+        ```typescript
+          <rapid-date-picker :disabledDaysOfTheWeek="${() => [1]}">
+        ```
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Component Methods
+
+This component doesn't have any methods.
+
+### Slots
+
+| Slot | Use |
+|---|---|
+| end | The icon in the calendar toggle. |
+| bottom | The bottom of the calendar. |
+
+### Parts
+
+| Part | Use |
+|---|---|
+| date-toggle | The calendar toggle element which shows the current value and toggle icon |
+| input-icon-container | The icon in the calendar toggle |
```

Comment:
Nitpick but you've not added full stops here, but you've got them on the table above